configure.ac

```diff
@@ -8321,6 +8321,12 @@ AC_ARG_ENABLE([sys-ca-certs],
     [ ENABLED_SYS_CA_CERTS=yes ]
     )
 
+AC_ARG_ENABLE([x9-146],
```

Comment:
Please refactor to use meaningful name for configure option and macro. Something like dual-signed / DUAL_SIGNED.

Reply:
`WOLFSSL_DUAL_ALG_CERTS`